addons/website/models/website.py

```diff
@@ -107,6 +107,8 @@ def _default_logo(self):
     partner_id = fields.Many2one(related='user_id.partner_id', relation='res.partner', string='Public Partner', readonly=False)
     menu_id = fields.Many2one('website.menu', compute='_compute_menu', string='Main Menu')
     homepage_id = fields.Many2one('website.page', string='Homepage')
+    custom_code_head = fields.Text('Custom <Head> code')
+    custom_code_footer = fields.Text('Custom <Footer> code')
```

Comment:
not really, imo it will be never display except in editor and with the t-raw.
So instead to add widget="text" when you want to edit it from the backend, it is more easy to keep it as text field ;)